### PR TITLE
Reset map center/bounds in overview initialize

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
Map view did not reset when navigating navigation -> overview, now it does.

File: src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
     /* ngInject */
     function OverviewController($scope, $stateParams, $q, $geolocation, leafletData,
-                                Directions, Map, MapControl, MapStyle, NavbarConfig, Notifications) {
+                                Config, Directions, Map, MapControl, MapStyle, NavbarConfig, 
+                                Notifications) {
```
Comment (moradology) [inline src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js:14]: :warning:  That's a lot of injections.

**Comment by moradology** [comment]
Pretty straightforward changeset... A little unclear to me why it didn't work before these changes - anyway, +1

File: src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js:14]: Yeah, its getting up there. Unfortunately this is the view that has a bit of everything.

**Comment by CloudNiner** [comment]
It didn't work before because the Config was copied by reference to Map.center and then ctl.map.center.

So, when the navigation view modifies ctl.map.center, it's actually modifying Config.center, because all of the other assignments are by reference.

**Pull request closed without merging**